src/PrestaShopBundle/Entity/Repository/LogRepository.php

```diff
@@ -206,6 +209,8 @@ public function getSearchQueryBuilder(SearchCriteriaInterface $searchCriteria)
      */
     public function getCountQueryBuilder(SearchCriteriaInterface $searchCriteria)
```

Comment:
@PululuK `@depracated` + `@see` or trigger error would be good enough? I honestly don't know 🤔

Reply:
Done ! thanks man